### ISSUE TITLE
Reranking network

### ISSUE THREAD

**Issue by yejy53** [ShihaoShao-GH/SuperGlobal#13]
Thank you for the great work you do. It seems that your reranking network does not require training, and there is no corresponding loss. I used the same processing logic. It seems that the reranking network for my extracted global features is not very ideal.

**Comment by ShihaoShao-GH** [comment]
You can finetune the hyper-parameters for reranking on a small validation datasets which can further boost the performance.
